src/quo2/components/tags/status_tags.cljs

```diff
@@ -45,49 +45,74 @@
          [icon/icon
           icon
           {:no-color true
-           :size     12}]]
+           :size     (if (= :large size) 20 12)}]]
         [text/text
          {:size   paragraph-size
           :weight :medium
           :style  {:padding-left 5
                    :color        text-color}} label]]])))
 
 (defn- positive
-  [size theme label]
+  [size theme label blur?]
   [base-tag
    {:size             size
-    :icon             :verified
+    :icon             :i/positive-state
     :background-color colors/success-50-opa-10
     :border-color     colors/success-50-opa-20
     :label            label
-    :text-color       (if (= theme :light)
-                        colors/success-50
-                        colors/success-60)}])
+    :text-color       (cond
+                        ;; The positive tag uses the same color for light themed and dark blur variant
+                        (or (= theme :light) blur?) colors/success-50
+                        (= theme :dark)             colors/success-60)}])
 
 (defn- negative
-  [size theme label]
+  [size theme label blur?]
   [base-tag
    {:size             size
-    :icon             :untrustworthy
+    :icon             :i/negative-state
     :background-color colors/danger-50-opa-10
     :border-color     colors/danger-50-opa-20
     :label            label
-    :text-color       (if (= theme :light)
-                        colors/danger-50
-                        colors/danger-60)}])
+    :text-color       (cond
```

Comment:
this condition looks strange, if its not light it will dark right ? so or blur is never happen, but what about light blur ?

Reply:
The `status-tag` component supports only  `light`, `dark`, and `dark blur`.  It does NOT have a `light blur` variant.

The `text-color` for `dark` and `dark blur` is the same for `negative` status tags. Hence, the condition.

Reply:
so (or blur? just redundant? it could be just 
```clojure
(if (= theme :light) colors/danger-50 colors/danger-60)
```

Reply:
Thought the `cond` would be clear to us when we look back later. Never mind, I will simply it using `if`.